### PR TITLE
Add support of "score" report item

### PR DESCRIPTION
**Pull request by pjanik**
[#182623252]

This PR is related to the new Interactive API: https://github.com/concord-consortium/lara/pull/982

It adds support of optional "score" report item. Report will ask interactive for "compactAnswer" items only when interactive provides report item metadata with `compactAnswerReportItemsAvailable=true`. This is done that way so we don't force older interactives like Sage or Geocode to render their report items 20-30 times (for each student) for no reason (they don't provide scores anyway). 

Once this approach is approved, I'll add more tests to increase the coverage.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,7 +64,6 @@
   "devDependencies": {
     "@cypress/code-coverage": "^3.9.7",
     "@istanbuljs/nyc-config-typescript": "^1.0.1",
-    "@mermaid-js/mermaid-cli": "^8.8.4",
```
Comment (scytacki) [inline package.json:67]: Was mermaid causing problems? I believe it is needed to run `npm run doc:diagrams`

**Review by scytacki — CHANGES_REQUESTED** [review]
Looks good. 

Only issue I found was the removal of mermaid.

File: package.json (hunk 1 above)
Comment (pjanik) [reply package.json:67]: Hm, I think I had some problems installing it locally, but when I added it back, it worked. Maybe some minor version upgrade helped.

**Comment by pjanik** [comment]
@scytacki, mermaid is added back, and I've also added a few tests that focus on the most important logic - when we should issue "getReportItemItem" iframe phone call and when we should skip it.

**Review by scytacki — APPROVED** [review]
looks good to me

**Pull request merged**